examples/rmg/commented/input.py

```diff
@@ -150,6 +150,16 @@
 )
 
 options(
+    	#provides a name for the seed mechanism produced at the end of an rmg run default is 'Seed'
+    name='RMGButane',	
```

Comment:
Perhaps change this to `libraryName` or `seedName` so it's easily understood and not too generic?
BTW, the terminology `seed` should perhaps be reconsidered since we're saving a library which the user can use either as a library or as a seed.